www.openfisca.fr/www.openfisca.fr.conf

```diff
@@ -0,0 +1,63 @@
+server {
```

Comment:
Has we changed anything to this file 🤔  ?

Reply:
Nothing changed.